docs/CHANGELOG.rst

```diff
@@ -43,6 +43,8 @@ Added
 
 Fixed
 -----
+-Fix ``macs2-callpeak`` to prevent the extension of intervals beyond chromosome
```

Comment:
This line is not formatted correctly. I think it needs a whitespace after the dash.